flaskapp/routes.py

```diff
@@ -62,16 +63,12 @@ def logout():
 @app.route("/account", methods=['GET', 'POST'])
 @login_required
 def account():
-    form = UpdateAccountForm()
+    form = UpdateAccountForm(obj=current_user)
     if form.validate_on_submit():
-        current_user.username = form.username.data
-        current_user.email = form.email.data
+        form.populate_obj(current_user)
```

Comment:
same as populating form data by using this: `Form(obj=instance)`, you can populate the instance with the form data like this: `form.populate_obj(instantiated_user)`